Function case (#317)
zowe/zowe-client-python-sdk: CHANGELOG.md

- Add type annotations for all methods [#280] (https://github.com/zowe/zowe-client-python-sdk/issues/280)

> - *Breaking*: Revised function names in `Logger` class and `files` class into snake_case. Supported for function case enforcer [#315] (https://github.com/zowe/zowe-client-python-sdk/issues/315)

Reviewer: Thoughts on making the changelog entry a bit more explicit for developers?

```suggestion
- *Breaking*: Revised function names in `Logger` class and `files` class into snake_case. Enabled pylint rule to enforce function case. [#315] (https://github.com/zowe/zowe-client-python-sdk/issues/315)
```
